core/node/da_dispatcher/src/da_dispatcher.rs

```diff
@@ -255,7 +255,7 @@ where
                 retries += 1;
                 let sleep_duration = Duration::from_secs(backoff_secs)
                     .mul_f32(rand::thread_rng().gen_range(0.8..1.2));
-                tracing::warn!(%err, "Failed DA dispatch request {retries}/{max_retries} for batch {batch_number}, retrying in {} milliseconds.", sleep_duration.as_millis());
+                tracing::warn!(%err, "Failed DA dispatch request {retries}/{} for batch {batch_number}, retrying in {} milliseconds.", max_retries+1, sleep_duration.as_millis());
```

Comment:
Nit: May make sense to format a macro in a multi-line fashion so that it's more readable.